docs/cli.rst

```diff
@@ -15,14 +15,26 @@ password to access the Copernicus Open Access Hub.
 
   sentinelsat -u <user> -p <password> -g <search_polygon.geojson>
 
-For convenience and added security you can set the username, password and DHuS URL as environment variables and omit their entry on the command line.
+For convenience and added security, there are two ways you can store your credentials and omit them from the command line call. You can set username, password and DHuS URL as environment variables.
```

Comment:
It would be great if we can include, which method takes precedence if both env-vars and a .netrc file are in place.